packages/environment-check/src/checks/install.ts

```diff
@@ -0,0 +1,64 @@
+import { existsSync, promises } from 'fs';
+import { t } from '../i18n';
+import { join } from 'path';
+import { spawnCommand, npmCommand } from '../command';
+
```

Comment:
Can we use format
[export] function <FUNCTION_NAME>() {
<FUNCTION_BODY>
}
for top level functions to be consistent with the rest of this module? 
Usually we use arrow function expressions for anonymous functions or functions that do not require a body.

Additionally, the name `install.ts` is a bit misleading.

Reply:
Updated